SUILesson2/SUILesson2/ContentView.swift

```diff
@@ -7,18 +7,303 @@
+                    Spacer()
+                }
+                startScreenYellowView
+                wellcomeTextView
+            }
+            showSettingsToggleView
+        }.animation(.spring())
+    }
+
+    // MARK: - private properties
+
+    private var startScreenYellowView: some View {
+        RoundedRectangle(cornerRadius: 10).fill(Color.yellow).offset(x: isOnToggle ? 500 : 0)
+    }
+
+    private var showSettingsToggleView: some View {
+        Toggle(isOn: $isOnToggle) {
+            Text(Constants.showSettingsName)
+        }.padding()
+    }
+
+    private var wellcomeTextView: some View {
+        Text(Constants.wellcomeName)
+            .offset(x: isOnToggle ? 500 : 0)
+            .frame(width: 170, height: 30)
+            .background(Color.white)
+            .cornerRadius(10)
+    }
+
+    private var sendMoneyImage: some View {
+        Image(systemName: Constants.addBillName)
+            .frame(width: 40, height: 40)
+            .background(Color.yellow)
+            .foregroundColor(.white)
+            .cornerRadius(5)
+            .padding()
+    }
+
+    private var titleView: some View {
+        HStack {
+            Spacer(minLength: 125)
+
+            Text(Constants.walletName)
+                .cornerRadius(10)
+                .padding()
+                .font(.title)
+                .italic()
+
+            Spacer()
+
+            Button {
+
+            } label: {
+                Image(systemName: Constants.plusImageName)
+                    .frame(width: 30, height: 30)
+                    .background(Color.green)
+                    .foregroundColor(.white)
+                    .cornerRadius(5)
+                    .padding()
+            }
+        }
+    }
+
+    private var debetCardView: some View {
+        VStack(spacing: 20) {
+            Text(Constants.debetCardName)
+            Text(debetCard.name)
+            HStack() {
+                Text(Constants.balanceName)
+                Text("\(debetCard.value)")
+                Text(Constants.rublesName)
+            }
+        }
+    }
+
+    private var addMoneyButtonView: some View {
+        Button(Constants.addMoneyName) {
+            self.isAddMoneyAlertShown = true
+        }
+        .alert(Constants.addMoneyName, isPresented: $isAddMoneyAlertShown, actions: {
+            TextField(Constants.valueOfMoneyName, text: Binding(get: {
+                self.addMoneyText
+            }, set: { newValue in
+                self.addMoneyText = newValue
+            }), onEditingChanged: { isEditingEnd in
+                guard
+                    !isEditingEnd,
+                    let money = Int(addMoneyText)
+                else { return }
+                self.debetCard.value += money
+                self.isAddMoneySuccessfullyAlertShown = true
+            })
+            .foregroundColor(.black)
+        })
+        .buttonStyle(BorderlessButtonStyle())
+        .padding()
+        .frame(width: 200, height: 40)
```

Comment:
```suggestion
        .frame(width:, height:)
```
в константы и везде по проекту также